schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -241,15 +241,17 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/automationAccounts_connections" },
                   { "$ref": "https://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/automationAccounts_variables" },
                   { "$ref": "https://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/automationAccounts_schedules" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2015-10-31/Microsoft.Automation.json#/resourceDefinitions/automationAccounts_jobs" },
```

Comment:
>automationAccounts_jobs [](start = 131, length = 23)

why are these removed?

Reply:
as long as the old api-version is still supported, I think you should keep them in schema.

Reply:
Understood. Fixed, and send a new commit.